spec/dataframe_spec.rb

```diff
@@ -934,6 +934,106 @@
       end
     end
   end
+  
+  context "#vector.at" do
```

Comment:
I thought we had discussed to keep column postional access with `df.at` and not bring the `vector` keyword anywhere into the fray.

Change this. `DataFrameByVector` need not exist. The default the way to access a vector is call methods on the dataframe directly. Specialized access is via rows, which is why keeping `DataFrameByRow` is acceptable.

See my comment here: https://github.com/v0dro/daru/pull/134#issuecomment-222455578